index.html

```diff
@@ -144,7 +144,7 @@ <h3 class="new">Profesional Art<br>Printing Data</h3>
                                 <li><a href="@" class="btn btn-car">Ruby</a></li>
                             </ul>
                     </div>
-                    <button type="button" class="card-btn">See Project</button>
+                    <button type="button" class="card-btn" id = "1">See Project</button>
```

Comment:
- [ ] To make sure we keep it as it is in the Figma file and greatly improve the UX of the users, Let's make sure we align the buttons with the image.
## Figma Design:point_down:
![image](https://user-images.githubusercontent.com/45668340/147251140-b3de19b1-a0d3-42fd-b2d0-a124c5abb0f2.png)
## Your Design:point_down:
![image](https://user-images.githubusercontent.com/45668340/147251273-8e6d34bd-72d8-4010-9286-bcc1461bac1b.png)

Reply:
Changes made...Thank you